### DISCUSSION TITLE
Role-based peer-learning opportunities

### DISCUSSION THREAD

**Discussion by chadkoh** [memser-spaceport/pl-network-hub#5] [Ideas]
**please describe your idea**
Create opportunities for PLN members from different companies connect based on their ROLE (ie COOs with COOs, marketers with marketers, etc) to peer learn, collaborate, and also build intranetwork ties.

**please make a case for this idea. What value would this add to the network? Would something become better or easier if this was implemented?**
We have the https://www.plnetwork.io/directory/ and so can piecemeal reach out and create _ad hoc_ partnerships. The next level would be creating events or spaces where people of different roles could take time to interact with one another. Network Services could facilitate this in a variety of ways, effectively creating an onramp for new connections. As we build relations within the network, not only can we can level up individuals through peer learning and mentoring, we can foster relations amongst member orgs to make the network as a whole _stronger_.

**Comment by chadkoh** [comment]
As a first step I would really like to see a needs assessment. Every individual I speak to is "Yes! This would be great!" but I would like to see if there is truly any demand for such opps.

**Reply by pandeyanujk** [reply]
Thanks for putting this idea through @chadkoh! There is a similar idea floating around in some of my conversations with the members too. Let me tag in the @memser-spaceport/engagement-wg so they can follow the conversation. Feel free to grab my office hours from the directory if you want to discuss this idea further.
